From: Ben Hutchings <ben@decadent.org.uk>
Date: Fri, 23 Oct 2015 00:51:44 +0100
Subject: target: Fix ABI change in 4.2.4
Forwarded: not-needed

Restore the now-redundant iscsi_np::np_ip field to avoid an ABI change.

--- a/include/target/iscsi/iscsi_target_core.h
+++ b/include/target/iscsi/iscsi_target_core.h
@@ -776,6 +776,7 @@ struct iscsi_np {
 	enum iscsi_timer_flags_table np_login_timer_flags;
 	u32			np_exports;
 	enum np_flags_table	np_flags;
+	unsigned char		np_ip[IPV6_ADDRESS_SPACE]; /* now unused */
 	u16			np_port;
 	spinlock_t		np_thread_lock;
 	struct completion	np_restart_comp;
